linear_feedback_controller/pd_plus_demo.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python
+
+"""
+Test PD+gravity tracking sinusoidal joint trajectory for Panda robot (7Dof).
+"""
+
+
+import numpy as np
+import pinocchio as pin
+
+import rclpy
+from rclpy.node import Node
+from rclpy.qos import QoSProfile, DurabilityPolicy, ReliabilityPolicy
+import rclpy.duration
+import rclpy.time
+from std_msgs.msg import String
+from linear_feedback_controller_msgs.msg import Sensor, Control, Pose, Twist
+from linear_feedback_controller_msgs_py.numpy_conversions import sensor_msg_to_numpy
+import linear_feedback_controller_msgs_py.lfc_py_types as lfc_py_types
+
+Ndof = 7
+
+AMPLITUDE = np.zeros(Ndof)
+AMPLITUDE[0] = 0.1
+
+PERIOD = 4.0 * np.ones(Ndof)
+
+
+def compute_sinusoid_q_delta_reference(amp, period, dt):
+    """
+    Compute a delta joint sinusoid reference with zero initial velocity.
+
+    a and c obtained for each joint using constraints:
+    delta_q(t=0.0) = 0
+    delta_q(t=period/2) = amp
+    """
+
+    w = 2 * np.pi / period
+    a = -amp
+    c = amp
+
+    delta_q = c + a * np.cos(w * dt)
+    dq = -w * a * np.sin(w * dt)
+
+    return delta_q, dq
+
+
+class PDPlusController(Node):
+    def __init__(self):
+        super().__init__("pd_plus_demo")
+        self.get_logger().warn("INITIALIZING pd_plus_demo")
+        self.publisher_control_ = self.create_publisher(
+            Control,
+            "/control",
+            QoSProfile(
+                depth=10,
+                durability=DurabilityPolicy.TRANSIENT_LOCAL,
+                reliability=ReliabilityPolicy.BEST_EFFORT,
+            ),
+        )
```

Comment:
```suggestion
        self.publisher_control_ = self.create_publisher(
            Control,
            "/control",
            qos_profile=qos_profile_system_default,
            qos_overriding_options=QoSOverridingOptions.with_default_policies(),
        )
```
`TRANSIENT_LOCAL` is not right QoS for this topic. This code will also let us later reconfigure the QoS in fly .
It also requires those imports
```python
from rclpy.qos import qos_profile_system_default
from rclpy.qos_overriding_options import QoSOverridingOptions
```